### PR TITLE
Dlg cli plot

### PR DESCRIPTION
**Pull request by lilyclements**
@africanmathsinitiative/developers This is ready for review.

Just as a side note - this is another dialog where it wasn't initially complete. I added in the newest code update (such as SetRCodeForControls etc) and updated the controls to the new ucr controls.

There are many text boxes in this dialog still as I wasn't sure of the long term plan for this dialog - that is, if these are going to change to receivers, combo boxes, inputs, etc.

I've updated the code so it works as it did before, however.

### CODE REVIEW

File: instat/dlgCliPlot.vb [hunk 1]
```diff
@@ -1,64 +1,77 @@
...
-        autoTranslate(Me)
-        ucrBase.clsRsyntax.SetFunction(frmMain.clsRLink.strClimateObject & "$cliplot")
-        ucrBase.clsRsyntax.iCallType = 0
-    End Sub
-
-
-
-    Private Sub chkStationName_Leave(sender As Object, e As EventArgs) Handles chkStationName.Leave
+Public Class dlgCliPlot
+    Public bFirstLoad As Boolean = True
```
Comment (Lunalo) [inline instat/dlgCliPlot.vb]: This should be Private since it is only being used in this class

File: instat/dlgCliPlot.vb [hunk 2]
```diff
@@ -1,64 +1,77 @@
...
+            SetDefaults()
+        End If
+        SetRCodeForControls(bReset)
+        bReset = False
+        ReopenDialog()
+        TestOkEnabled()
+    End Sub
+
+    Private Sub InitialiseDialog()
+        ucrBase.clsRsyntax.iCallType = 0
```
Comment (Lunalo) [inline instat/dlgCliPlot.vb]: I am not sure why we need this ICallType here

**Comment by Lunalo** [comment]
Also The events need to be updated

**Comment by lilyclements** [comment]
@Lunalo this is ready for a re-review

**Comment by Lunalo** [comment]
Hi @lilyclements 

Can you find time to replace textboxes with ucrInnputTextboxes before I review ?

**Pull request closed without merging**